plugins/chat/app/serializers/chat/thread_serializer.rb

```diff
@@ -31,7 +35,7 @@ def reply_count
     end
 
     def include_preview?
-      @opts[:include_preview]
+      @opts[:include_thread_preview]
```

Comment:
There are still a few places using `include_preview: true` which is making some things (like the thread list) a bit wonky:

![image](https://github.com/discourse/discourse/assets/920448/60bcb51e-e352-4913-be67-e0f2813372e1)

https://github.com/discourse/discourse/blob/fda4873ef243a672b318e6c77099b27a9118afa2/plugins/chat/app/serializers/chat/thread_list_serializer.rb#L13-L13

https://github.com/discourse/discourse/blob/fda4873ef243a672b318e6c77099b27a9118afa2/plugins/chat/app/controllers/chat/api/channel_threads_controller.rb#L36-L36

https://github.com/discourse/discourse/blob/fda4873ef243a672b318e6c77099b27a9118afa2/plugins/chat/app/serializers/chat/view_serializer.rb#L16-L16

Though the whole ViewSerializer file can be deleted since you are not using it anymore :)

Reply:
I did notice though we don't seem to set up these meta options for the channel anywhere anymore:

https://github.com/discourse/discourse/blob/fda4873ef243a672b318e6c77099b27a9118afa2/plugins/chat/app/serializers/chat/view_serializer.rb#L58-L68

They probably need to be moved to `ChannelSerializer`